MediaGalleryUi/view/adminhtml/web/js/form/element/ui-select.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'Magento_Ui/js/form/element/ui-select'
+], function (uiSelect) {
+    'use strict';
+
+    return uiSelect.extend({
+        /**
+         * Get path to current option
+         *
+         * @param {Object} data - option data
+         * @returns {String} path
+         */
+        getPath: function (data) {
```

Comment:
Can we avoid the need to extend the component by using showPath=false configuration and just using a custom template